src/layout/Grid.js

```diff
@@ -137,16 +126,238 @@
+        var handleSize = this._dragCell.handleSize();
+        
+        //Determines which edge cell (if any) this._currLoc is hovering over
+        //An "edge" meaning a dropCell on the exterrior edge of a surface that covers many cells
+        var onSouthEdge = this._dragCell.gridRowSpan() === this._currLoc[1] - this._dragCell.gridRow() + 1;
+        var onNorthEdge = this._dragCell.gridRow() === this._currLoc[1];
+        var onEastEdge = this._dragCell.gridColSpan() === this._currLoc[0] - this._dragCell.gridCol() + 1;
+        var onWestEdge = this._dragCell.gridCol() === this._currLoc[0];
+        
+        var top = this._offsetY + ((this._currLoc[1]) * this._rowSize);
+        var left = this._offsetX + ((this._currLoc[0]) * this._colSize);
+        var width = this._colSize - this.gutter();
+        var height = this._rowSize - this.gutter();
+        
+        var handlesInRange = [];
+        if(top + height >= e.clientY && top + height - handleSize <= e.clientY && onSouthEdge){
+            handlesInRange.push('s');//within SOUTH handle range
+        }
+        else if(top <= e.clientY && top + handleSize >= e.clientY && onNorthEdge){
+            handlesInRange.push('n');//within NORTH handle range
+        }
+        if(left + width >= e.clientX && left + width - handleSize <= e.clientX && onEastEdge){
+            handlesInRange.push('e');//within EAST handle range
+        }
+        else if(left <= e.clientX && left + handleSize >= e.clientX && onWestEdge){
+            handlesInRange.push('w');//within WEST handle range
+        }
+        if(handlesInRange.length){
+            handle = handlesInRange.join('');
+        }
+        return handle;
+    };
+    
+    
+    
+    var hideDragHandles = function(){
+        var handles = document.getElementsByClassName('dragHandle');
+        for(var i in handles){
+            if(handles[i] !== null && typeof (handles[i]) !== 'undefined'){
+                if(handles[i].style !== null && typeof (handles[i].style) !== 'undefined'){
+                    handles[i].style.visibility = 'hidden';
+                }
+            }
+        }
+    };
+    
+    var showDragHandles = function(){
+        var handles = document.getElementsByClassName('dragHandle');
+        for(var i in handles){
+            if(handles[i] !== null && typeof (handles[i]) !== 'undefined'){
+                if(handles[i].style !== null && typeof (handles[i].style) !== 'undefined'){
+                    handles[i].style.visibility = 'visible';
+                }
+            }
+        }
+    };
+    
+    Grid.prototype.createDropTarget = function (loc) {
+        var context = this;
+        var col = loc[0] - this._dragCellOffsetX;
+        var row = loc[1] - this._dragCellOffsetY;
+        var colSpan = this._dragCell.gridColSpan();
+        var rowSpan = this._dragCell.gridRowSpan();
+        
+        var dropTarget = document.createElement('div');
+        dropTarget.id = 'grid-drop-target';
```

Comment:
Given the lifetime of the grid-drop-target this is probably fine.  But for the future you should (for safety) include the ```this.id()``` in the id to avoid ID clashes.

Also I would be inclined to append these DIVs within ```this._target``` rather onto the page in general.